README.md

```diff
@@ -82,6 +82,25 @@ GetGroupsFilter getGroupsFilter = new GetGroupsFilter().withOffset(0)
 List<Group> groupsList = groups.getGroups(getGroupsFilter);
 ```
 
+#### Retrieving data about commands
+`LOApiClient` allows retrieve data about a single command or list of commands. To retrieve the data about commands from Live Objects, you can use the sample code:
+
+Getting commands client:
+```java
+DeviceManagement deviceManagement = client.getDeviceManagement();
+Commands commands = deviceManagement.getCommands();
```

Comment:
minor : while reading that part of code, I was confuse on getting `commands` right here. Maybe it's a wording issue and `commandManagement` would have been less confusing (at least for me).

Reply:
I agree that it can be confusing but we take that names from Live Objects Rest Api and their naming convention in swagger.
`Device management - Inventory` so we have class `Inventory` for managing devices, `Device management - Groups` so we have `Groups` class for managing groups and finally `Device management - Commands` so we have `Commands` class. But this approach can introduce a misunderstanding as you mentioned. We will look closer to this issue in the nearest future and maybe we make some refactor (renaming).

Reply:
I understand, no worry, that's fine ; thx for this feedback 👍